docs/developers_guide/command_line.md

```diff
@@ -196,7 +196,7 @@ directory:
 
 ```none
 polaris serial [-h] [--steps STEPS [STEPS ...]]
-                 [--no-steps NO_STEPS [NO_STEPS ...]]
+                 [--skip-steps SKIP_STEPS [SKIP_STEPS ...]]
```

Comment:
```suggestion
                 [--skip_steps SKIP_STEPS [SKIP_STEPS ...]]
```
@altheaden, I realize this is nothing you introduced but let's go with underscores instead of hyphens for command-line flags (here and elsewhere in the PR) for better consistency with the rest of polaris.